fineract-provider/src/main/java/org/apache/fineract/infrastructure/configuration/service/ExternalServicesPropertiesReadPlatformServiceImpl.java

```diff
@@ -137,6 +138,23 @@ public MessageGatewayConfigurationData extractData(final ResultSet rs) throws SQ
         }
     }
 
+    private static final class MasivianConfigurationDataExtractor implements ResultSetExtractor<MasivianConfigurationData> {
+
+        @Override
+        public MasivianConfigurationData extractData(final ResultSet rs) throws SQLException, DataAccessException {
+            String smsApiUrl = null;
+            String smsAuthorization = null;
+            while (rs.next()) {
+                if (rs.getString("name").equalsIgnoreCase(ExternalServicesConstants.MASIVIAN_SMS_API_URL)) {
+                    smsApiUrl = rs.getString("value");
+                } else if (rs.getString("name").equalsIgnoreCase(ExternalServicesConstants.MASIVIAN_SMS_API_AUTHORIZATION_HEADER)) {
```

Comment:
same as this . perhaps , You can have a variable for rs.getString . instea dof replicating